common/retry_delays_config.go

```diff
@@ -0,0 +1,73 @@
+package common
+
+import (
+	"fmt"
+
+	commontypes "github.com/agglayer/aggkit/common/types"
+	"github.com/agglayer/aggkit/config/types"
+)
+
+var _ commontypes.RetryPolicyConfigurer = (*RetryDelaysConfig)(nil)
+
+// RetryDelaysConfig defines the configuration for retry delays.
+type RetryDelaysConfig struct {
+	// MaxRetries is the maximum number of retries to attempt.
+	// if MaxRetries is -1, it means infinite retries.
+	// if MaxRetries is 0, it means no retries will be attempted.
+	MaxRetries int `mapstructure:"MaxRetries"`
+	// Delays is a list of durations to wait before each retry.
+	// if there are more attempts that item o the list is used the last one
+	Delays []types.Duration `mapstructure:"Delays"`
```

Comment:
Since it is not unmarshalled from the TOML, I beleive the annotations can be removed?
```suggestion
	MaxRetries int
	// Delays is a list of durations to wait before each retry.
	// if there are more attempts that item o the list is used the last one
	Delays []types.Duration
```